netmap/netmap_contract.go

```diff
@@ -256,6 +256,9 @@ func AddPeer(nodeInfo []byte) {
 	}
 
 	addToNetmap(ctx, candidate)
+	if notaryDisabled {
```

Comment:
Success notifications are useful for both notary enabled and disabled environments, no?